package.json

```diff
@@ -192,7 +192,7 @@
             "editor/context": [
                 {
                     "command": "clangd.typeHierarchy",
-                    "when": "resourceLangId == cpp",
+                    "when": "resourceLangId == cpp && extension.vscode-clangd.serverRunning",
```

Comment:
This reminds me of a point I have missed in the previous review (sorry) -- we should only enable the typeHierarchy when the server capability `typeHierarchyProvider` is true, I think this should not be a big problem in practice (as typeHierarchy is implemented since clangd 9).

If the fix is trivial, would be nice to fix that -- I think we probably need a context value like `extension.vscode-clangd.hastypeHierarchyProvider`.

Reply:
Added a check for `typeHierarchyProvider` in the updated patch.